CIP-SUBLEDGER/README.md

```diff
@@ -0,0 +1,144 @@
+---
+CIP: ?
+Title: A Subledger in Plutus Enabling Native-Asset Seizure for Regulatory Compliance
+Status: Proposed
+Category: Ledger
+Authors:
+    - Micah Kendall <micah@butane.dev>
+Implementors: [Micah Kendall, <you?>]
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/?
+Created: 2024-05-30
+License: CC-BY-4.0
+---
+
+# CIP \*\*\*: A Subledger in Plutus Enabling Native-Asset Seizure for Regulatory Compliance
+
+## Abstract
+We aim to establish a simple standard to ensure native assets can exist in a locked smart contract system, referred to as the "subledger". This subledger will enable functionality defined in the minting policy to seize assets.
+
+This design preserves the benefits of Cardano's native assets by enabling the reuse of most EUTxO patterns. It is significant that DEXs or other Plutus applications can be easily adapted to function within the subledger environment.
+
+# Motivation: why is this CIP necessary?
```

Comment:
_This_ is the section there "Regulatory Compliance" should be mentioned as a principal motivation, rather than in the CIP title.